integration/time_drift/time_drift.bats

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will make sure that guest time
+# is synced with the host
+
+set -e
+
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+# Save guest time
+GUEST_TMP_FILE=$(mktemp guest.XXXXX)
+# Save host time
+HOST_TMP_FILE=$(mktemp host.XXXXX)
+
+function remove_tmp_files() {
+	rm "${GUEST_TMP_FILE}" "${HOST_TMP_FILE}"
+}
+
+trap remove_tmp_files EXIT
+
+setup() {
+	clean_env
+
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "Verify guest time is synced with the host" {
+	CONTAINER_NAME="test"
+	IMAGE="ubuntu"
```

Comment:
Can we use a smaller image that we may already have loaded for the tests here, such as `busybox` or `alpine`. I think the `date` from busybox for instance will be accurate and featureful enough for this test?
We are on a drive to reduce the time and resource consumption of our CI - part of that is trying to use, and re-use, the smallest container images we can for the tests. Otherwise they consume significantly more download and disk bandwidth, which also takes longer to grab.